packages/Popover/src/components/Content/Content.js

```diff
@@ -79,6 +79,20 @@ const Content = React.forwardRef((props, ref) => {
     width: content.width,
   };
 
+  let hasCardComponent = false;
+  const popoverContent = isEager
+    ? React.Children.map(props.children, child => {
+        if (child.type && child.type.displayName === "Popover.Card") {
+          hasCardComponent = true;
+          return React.cloneElement(child, {
+            id: content.ariaId,
+          });
+        }
+
+        return child;
+      })
+    : props.children;
+
```

Comment:
Currently in the dropdown component we are adding an id to the <Popover.content> directly in order to set focus on the Dropdown content. 
https://github.com/acl-services/paprika/blob/master/packages/DropdownMenu/src/DropdownMenu.js#L122

I believe with these changes we won't need to do that anymore in the dropdown component if it is done internally in the Popover right?